### DISCUSSION TITLE
Asset Media Card - Circle text

### DISCUSSION THREAD

**Discussion by pihcardozo** [Telefonica/mistica-design#1333] [Proposals]
### Goal

Incluir mais um asset no grupo de assets do componente Media Card. Precisamos entender se isso pode ser absorvido pelo DS Mística ou se será um asset local. Como também dar andamento no desenvolvimento do que já existe documentado, para possibilitar a inclusão do asset citado. 

### Context

Precisamos de um selo com texto para ser aplicado nos cards promocionais. Este selo levará informações de porcentagem de desconto.

### Impact

Ter mais um elemento para ser utilizado no layout do card, deixando a tag livre para outros fins. 

### Discussion Questions

O desenvolvimento destes assets no card pode ser priorizado?
Renomeei o asset como "Circle text", usando como referência os assets existentes.

### Do you have a visual proposal?

https://www.figma.com/file/Y89JWviWrWV4RFFyqwCj3b/Asset-Media-Card---Circle-text?type=design&node-id=110%3A14234&mode=design&t=4IvJbwJfmDCISNvH-1

**Comment by yceballost** [comment]
Buenas Fábio!

No vemos la necesidad de crear un componente Circle + Texto porque es algo que ya se puede hacer actualmente
https://tinyurl.com/2n9tmube

Lo que hay que evolucionar para que sea posible hacer lo que hay en el figma es habilitar el área de "asset" de la media card.

La definción y tickets para desarrollo ya están hechos por nuestra parte. Cread un ticket en el jira de Novo App y enlazad los tickets de web
https://jira.tid.es/browse/WEB-760
https://jira.tid.es/browse/ANDROID-11582
https://jira.tid.es/browse/IOS-8015
